Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -56,7 +56,7 @@ extension SentryFeedback: SentrySerializable {
         if let associatedEventId = associatedEventId {
             dict["associated_event_id"] = associatedEventId.sentryIdString
         }
-        dict["source"] = source.rawValue
+        dict["source"] = source.description
```

Comment:
`l`: Just want to point out that in most cases it is bad practice to use `description` in any serialization, because if the implementation of `description` in line 7 is removed, the Swift compiler could still synthesize the `CustomStringConvertible`, and it's not guaranteed to match the raw string value (could include information like `Optional<String>`)

Reply:
Good point, i'll rename it and avoid using CustomStringConvertible for this.

Reply:
Hmm, are we sure swiftc will synthesize CustomStringConvertible for an `@objc enum`? When I removed the conformance, here's how compilation broke:
<img width="715" alt="image" src="https://github.com/user-attachments/assets/7679834d-5f73-40e6-b1d9-f06048ef88d4" />

Reply:
I think you're thinking of NSObject descendants, since `- [NSObject description]` automatically gets inherited by them all?

Reply:
Out of an abundance of caution I will make the change, but I found this little exploration interesting so just wanted to share.